services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouterHttpRequestStats.java

```diff
@@ -159,8 +168,8 @@ public RouterHttpRequestStats(
     Rate tardyRequestRate = new OccurrenceRate();
 
     incomingRequestSensor = registerSensor("request", new Count(), requestRate);
-    incomingRequestSensorOtel = metricsRepository.getOpenTelemetryMetricsRepository()
-        .getCounter("incoming_call_count", "Number", "Count of all incoming requests");
+    incomingRequestSensorOtel =
+        (LongCounter) metricsRepository.getOpenTelemetryMetricsRepository().getInstrument(INCOMING_CALL_COUNT);
```

Comment:
The casting here is a bit unfortunate... I wonder if we could do better by adding a generic type param to `MetricEntity`?